target/linux/rockchip/Makefile

```diff
@@ -16,7 +16,7 @@ endef
 
 include $(INCLUDE_DIR)/target.mk
 
-DEFAULT_PACKAGES += uboot-envtools partx-utils e2fsprogs mkf2fs kmod-gpio-button-hotplug
```

Comment:
It would be better to paste the panic log here and see if there's any fix could apply.